src/plugins/compact-view/compact.css

```diff
@@ -0,0 +1,4 @@
+.compact-view-no-separating-lines.compact-view-highlight-alternating-lines
+  .dcg-expressionfolder:nth-child(2n of .dcg-expressionfolder) {
+  background-color: #d6d5f1 !important;
```

Comment:
I try to not use `!important` in general. Are you able to modify these (and other rules) to avoid `!important` where possible? Oftentimes just increasing the specificity a little fixes it.

Reply:
I've gotten rid of almost every `!important` by replacing it a repeated class selector (which seems to increase specificity). Still a bit cursed but less so I hope lol.